src/app/header/header.component.ts

```diff
@@ -20,11 +20,14 @@ export class HeaderComponent implements OnInit {
     private updateDataService: UpdateDataService,
     private route: Router,
     private articleDataSource: ArticleListDataService
```

Comment:
`articleDataSource` should be renamed to `articleDataService`

Reply:
**Changes Implemented**